src/Forms/ElementalAreaField.php

```diff
@@ -247,6 +248,9 @@ public function saveInto(DataObjectInterface $dataObject)
 
             $element->updateFromFormData($data);
             $element->write();
+            $elements[] = $element;
         }
+
+        $this->extend('onSaveInto', $elements, $dataObject, $elementData);
```

Comment:
just wondering it it makes sense to change the variable name `$elementData`, because there's a plural `$elements`, and a singlular `$elementData`

Would it make sense to rename it to `$elementalAreaFieldValue` to make it obvious what it is?

Reply:
Maybe, but then the loop above looks weird as you're iterating through `elementalAreaFieldValue` which doesn't seem like it would be an array unless you know the internals.

How about `$elementsChanged` and `$allElementData`?

Reply:
Ah yeah course, my bad, "data" is essentially a plural - calling it elementArrays instead of elementData is would be equivalent, though data sounds nicer

Change `$elements` to `$updatedElements` and I'm happy naming wise